### ISSUE TITLE
Accessibility does not work properly with nested ScrollViews

### ISSUE THREAD

**Issue by Cxxxx100** [facebook/react-native#14766]
### Is this a bug report?

Yes

### Have you read the Bugs section of the Contributing to React Native Guide?

Yes

### Environment

1. `react-native -v`: 0.45.1
2. `node -v`: v6.2.2
3. `npm -v`: 3.9.5
4. `yarn --version` (if you use Yarn): N/A

Then, specify:

1. Target Platform (e.g. iOS, Android): Android
2. Development Operating System (e.g. macOS Sierra, Windows 10): macOS
3. Build tools (Xcode or Android Studio version, iOS or Android SDK version, if relevant): N/A

### Steps to Reproduce

1. Turn on Accessibility / Screen reader on your device
2. Open the app: https://snack.expo.io/BJ9DuKbEb
3. Use right swipe gesture to navigate the page

### Expected Behavior

Accessibility content is read left to right, top to bottom, scrolling to read offscreen content

### Actual Behavior

Content is read starting with the second item in the horizontal scroll view, then the first, and skipping the last item.  When we get to the bottom of the screen vertical ScrollView is not scrolled and instead soft buttons at the bottom of the device are read

![1ro4z9](https://user-images.githubusercontent.com/22461323/27657454-9474ede0-5c01-11e7-8090-856eca00304b.gif) 

### Reproducible Demo

https://snack.expo.io/BJ9DuKbEb

**Comment by Cxxxx100** [comment]
I was playing around with this a bit more and resolved the issue with the horizontal ScrollViews.  I just needed to give the Text components the same margins.

However, the vertical ScrollView will still not scroll when I reach the bottom of the page.  See the below animated gif:

![1rofja](https://user-images.githubusercontent.com/22461323/27662560-1aa837bc-5c13-11e7-8a42-439dbb2c9f34.gif)

Here's a snack of the new code: https://snack.expo.io/SyQ8Osb4Z

I changed `marginBottom: 5` to `margin: 10` on the instructions style

**Comment by Cxxxx100** [comment]
Seems this issue is specific to FireOS.  I tested the following devices:

Nexus 5: ScrollView scrolls as expected
Samsung Galaxy S5: ScrollView scrolls as expected
Fire HD 6: ScrollView does NOT scroll as expected

**Comment by hramos** [comment]
Hi there! This issue is being closed because it has been inactive for a while. Maybe the issue has been fixed in a recent release, or perhaps it is not affecting a lot of people. Either way, we're automatically closing issues after a period of inactivity. Please do not take it personally!

If you think this issue should definitely remain open, please let us know. The following information is helpful when it comes to determining if the issue should be re-opened:

- Does the issue still reproduce on the latest release candidate? Post a comment with the version you tested.
- If so, is there any information missing from the bug report? Post a comment with all the information required by the [issue template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md).
- Is there a pull request that addresses this issue? Post a comment with the PR number so we can follow up.

If you would like to work on a patch to fix the issue, *contributions are very welcome*! Read through the [contribution guide](http://facebook.github.io/react-native/docs/contributing.html), and feel free to hop into [#react-native](https://discordapp.com/invite/0ZcbPKXt5bZjGY5n) if you need help planning your contribution.

**Comment by rkhatak** [comment]
Hi,

I am using android emulator nexus s API and here nested scrollview is not working.
Please let me know if any workaround.
